tornado/httpclient.py

```diff
@@ -110,9 +110,9 @@ class AsyncHTTPClient(Configurable):
 
         def handle_response(response):
             if response.error:
-                print "Error:", response.error
+                print("Error:", response.error)
```

Comment:
This one means something different on python 2 and python 3. To make sure that the same thing is printed in both versions, it should be something like `print("Error: %s" % response.error)`.